worldeditadditions/utils/selector_helps.lua

```diff
@@ -0,0 +1,12 @@
+-- Returns the player's facing direction on the horizontal axes only.
+-- @param	name	string	The name of the player to return facing direction of.
+-- @return	table	Returns axis name and sign multiplyer.
```

Comment:
Awesome description! This looks like a very useful function to have implemented.

Reply:
Yah. WE has a 3d get look direction function but that's not terribly useful when you want to know which way player is facing even when their looking up or down.